docs/guide/workflow.md

```diff
@@ -0,0 +1,91 @@
+
+``` yaml hl_lines="16 17 18 31 32 33 34 35"
+ezpublish:
+    system:
+        # Workflow configuration is SiteAccess-aware
+        default:
+            workflows:
+                # Identifier of the workflow
+                custom_workflow:
+                    name: 'Custom Workflow'
+                    matchers:
+                        # Which Content Types can use this workflow
+                        content_type: article
+                        # Which status of the Content item can use this workflow. Available statuses are draft and published.
+                        content_status: draft
+                    # All stages the content goes through
+                    stages:
+                        draft:
+                            label: 'Draft'
+                            color: '#f15a10'
+                        technical:
+                            label: 'Technical review'
+                            color: '#10f15a'
+                        proofread:
+                            label: 'Proofread'
+                            color: '#5a10f1'
+                        done:
+                            label: 'Done'
+                            color: '#301203'
+                    initial_stage: draft
+                    # Available transitions between stages
+                    transitions:
+                        to_technical:
+                            from: draft
+                            to: technical
+                            label: 'To technical review'
+                            icon: '/bundles/ezplatformadminui/img/ez-icons.svg#comment'
+                        back_to_draft:
+                            reverse: to_technical
+                            label: 'Back to draft'
+                            icon: '/bundles/ezplatformadminui/img/ez-icons.svg#comment'
+                        to_proofread:
+                            from: technical
+                            to: proofread
+                            label: 'To proofreading'
+                            icon: '/bundles/ezplatformadminui/img/ez-icons.svg#comment'
+                        back_to_technical:
+                            reverse: to_proofread
+                            label: 'Back to technical review'
+                            icon: '/bundles/ezplatformadminui/img/ez-icons.svg#comment'
+                        done:
+                            from: proofread
+                            to: done
+                            label: 'Done'
+                            icon: '/bundles/ezplatformadminui/img/ez-icons.svg#comment'
+```
+
+Each stage in the workflow has an identifier and can be assigned a label and a color (lines 14-16).
+
+Each transition also has an identifier. It must either state between which stages it transitions, or else be marked as `reverse` of a different transition.
```

Comment:
It must state between which stages it transitions, or it will be marked as `reverse` of a different transition.